src/schematools/types.py

```diff
@@ -149,6 +151,7 @@ def get_tables(
             tables += self.through_tables
         return tables
 
+    @lru_cache()
```

Comment:
Dit werkt niet goed op methods. Dan cache je namelijk ook `self` steeds.

Het enige wat werkt is in `__init__()` toevoegen:
`self.get_table_by_id = lru_cache()(get_table_by_id)`

Als je toch een decorator op de method wilt, is dit en mooie truc: https://gist.github.com/ar45/a08a05d34996d33e03391a81bfee3b1e

Reply:
Dit is niet de reguliere `lru_cache`.

Reply:
Maar deze: https://pypi.org/project/methodtools/

Reply:
Ga ik gebruiken als `@methodtools.lru_cache`, om verwarring te voorkomen.